javacutil/src/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -702,29 +704,27 @@ public static boolean isMethodInvocation(
 
     /**
      * Returns the ExecutableElement for a method declaration of methodName, in class typeName, with
-     * params parameters.
-     *
-     * <p>TODO: to precisely resolve method overloading, we should use parameter types and not just
-     * the number of parameters!
+     * params parameters. Errs if there is more than one matching method.
```

Comment:
Now that the TODO is resolved, how about adding a cross-reference to the method that takes the parameter types and explaining the trade-offs?